src/index.ts

```diff
@@ -46,16 +46,76 @@ export default class DataSource {
       .join('&');
 
     const url = `${this.sorcererUrlBase}/${this.key}?${paramStr}`;
+    const isTokenBuilder = options['body'] && JSON.parse(options['body']).tokenApiVersion;
```

Comment:
This may break if there is a body that is not JSON? We may want something like:

```suggestion
    let isTokenBuilder;
    try {
      isTokenBuilder = !!JSON.parse(options['body']).tokenApiVersion;
    } catch(_e) {
      isTokenBuilder = false;
    }
```